pkg/backend/config/config.go

```diff
@@ -24,39 +24,80 @@ import (
 )
 
 const (
+	IAMEnabledEnvVar = "IAM_ENABLED"
+
+	AuthOIDCIssuerEnvVar        = "AUTH_OIDC_ISSUER"
+	AuthOIDCClientIDEnvVar      = "AUTH_OIDC_CLIENT_ID"
+	AuthOIDCClientSecretEnvVar  = "AUTH_OIDC_CLIENT_SECRET" // #nosec G101
+	AuthOIDCTokenURLEnvVar      = "AUTH_OIDC_TOKEN_URL"     // #nosec G101
+	AuthOIDCIntrospectURLEnvVar = "AUTH_OIDC_INTROSPECT_URL"
+
+	AuthRoleSyncJwtRoleClaimEnvVar = "AUTH_ROLE_SYNC_JWT_ROLE_CLAIM"
+
+	AuthorizationRbacRuleFilePathEnvVar = "AUTHZ_RBAC_RULE_FILE_PATH"
+
 	BackendRestHost       = "BACKEND_REST_HOST"
 	BackendRestDisableTLS = "BACKEND_REST_DISABLE_TLS"
 	BackendRestPort       = "BACKEND_REST_PORT"
 	HealthCheckAddress    = "HEALTH_CHECK_ADDRESS"
 
+	DisableOrchestrator               = "DISABLE_ORCHESTRATOR"
+	OrchestratorAuthBearerTokenEnvVar = "ORCHESTRATOR_AUTH_BEARER_TOKEN"
+
+	UISitePath = "UI_SITE_PATH" // TODO: UI site should be moved out of the backend to nginx
+
 	DBNameEnvVar     = "DB_NAME"
 	DBUserEnvVar     = "DB_USER"
 	DBPasswordEnvVar = "DB_PASS"
 	DBHostEnvVar     = "DB_HOST"
 	DBPortEnvVar     = "DB_PORT_NUMBER"
 	DatabaseDriver   = "DATABASE_DRIVER"
 	EnableDBInfoLogs = "ENABLE_DB_INFO_LOGS"
+	FakeDataEnvVar   = "FAKE_DATA"
+	LocalDBPath      = "LOCAL_DB_PATH"
 
-	LocalDBPath = "LOCAL_DB_PATH"
+	LogLevel = "LOG_LEVEL"
+)
 
-	FakeDataEnvVar      = "FAKE_DATA"
-	DisableOrchestrator = "DISABLE_ORCHESTRATOR"
+type AuthenticationOIDC struct {
```

Comment:
Not sure this is the right place to define these. Maybe under iam/types?

Reply:
Resolved